### PR TITLE
Fixes #5 - block now delivers complex stream

### LINKED ISSUE #5: Produce Complex IQ instead of interleaved Float
It would be good if instead of producing a stream of floats that we then have to de-interleave, we just produce the stream of complex IQ samples.  This should hopefully be fairly straightforward and just be a matter of de-interleaving within the module itself.

### PR DESCRIPTION
**Pull request by nsbruce**
@cjam

### CODE REVIEW

**Comment by nsbruce** [comment]
Conflicts are turd-o. Closing and retrying.

**Comment by nsbruce** [comment]
Okay @cjam it's ready for you.

File: python/FlexSource.py [hunk 1]
```diff
@@ -27,17 +27,19 @@
 from flex import FlexApi
 from RingBuffer import RingBuffer
 
-
+# For debugging... Sick comment, Nick.
```
Comment (cjam) [inline python/FlexSource.py]: I think we moved this into the class right?

File: python/FlexSource.py [hunk 2]
```diff
@@ -188,14 +196,15 @@ def stop(self):
...
-    output buffer of the gnu block
+    we pull the appropriate amount of data off of the queue and push it into
+    the output buffer of the gnu block
     """
     def work(self, input_items, output_items):
         out = output_items[0]
         num_outputs = 0
         try:
             items = self.rx_buffer.get(out.size)
+            items = items.astype(numpy.float32).view(numpy.complex64)
```
Comment (cjam) [inline python/FlexSource.py:205]: :+1:

**Review by cjam — APPROVED** [review]
nice, this is great.  It's awesome that it was so simple.. in the sense of just changing the shape of the output.

File: python/FlexSource.py [hunk 3]
```diff
@@ -27,17 +27,19 @@
 from flex import FlexApi
 from RingBuffer import RingBuffer
 
-
+# For debugging... Sick comment, Nick.
+self._debug = False
```
Comment (cjam) [inline python/FlexSource.py]: This can be removed as it's now in the class

**Pull request merged**